tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -286,6 +286,23 @@ object TastyFormat {
   }
   object NameTags extends NameTags
 
+  def nameTagToString(tag: Int) = {
```

Comment:
Add a comment to say that this is here for debugging

Reply:
done, also documented that it is linked to the`NameTags` object